src/domain/interfaces/transaction-api.interface.ts

```diff
@@ -207,4 +207,8 @@ export interface ITransactionApi {
     messageHash: string;
     signature: string;
   }): Promise<unknown>;
+
+  clearMessagesBySafe(args: { safeAddress: string });
+
+  clearMessagesByHash(args: { messageHash: string });
```

Comment:
Missing explicit return type here.

Reply:
Added in [add1cee](https://github.com/safe-global/safe-client-gateway-nest/pull/654/commits/add1cee38699a4159183b097465a5e79e2889962)